docs/release-cadence.md

```diff
@@ -15,6 +15,6 @@ cf.gov will be on a two week release cadence that will line up with the platform
 
 Monday | Tuesday | Wednesday | Thursday | Friday
 ------ | ------- | --------- | -------- | ------
-| 5.1 branched, 5.1.0 released, deployed to beta | | Hotfix 5.1.1 committed, released, deployed to beta | |
-5.1.1 deployed to www and content | | Hotfix 5.1.2 committed, released, deployed to www and content | | | |
-| 5.2 branched, deployed to beta | |
+| | | 5.1 branched, 5.1.0 released, deployed to beta | Hotfix 5.1.1 committed, released, deployed to beta |
+| 5.1.1 deployed to www and content | | Hotfix 5.1.2 committed, released, deployed to www and content | | |
+| | | 5.2 branched, 5.2.0 released, deployed to beta |
```

Comment:
Interesting, mkdocs and GitHub seem to render the markdown for the table slightly differently. The table is correct for me locally.